samples/apps/copilot-chat-app/WebApp/src/components/chat/ChatRoom.tsx

```diff
@@ -81,8 +85,15 @@ export const ChatRoom: React.FC = () => {
             sender: account?.homeAccountId,
             content: value,
         };
+        setIsBotTyping(true);
         dispatch(updateConversation({ message: chatInput }));
-        await chat.getResponse(value, selectedId);
+        try {
+            await chat.getResponse(value, selectedId);
+        } catch (error) {
+            setIsBotTyping(false);
```

Comment:
nit: the line #93 and #96 can go to finally block of the try statement and catch block can be removed.

Reply:
Good catch!